### PR TITLE
up: Adapt for DPD France

### PR DESCRIPTION
**Pull request by Snowbaha**
Hello,
I adapted some stuff for France, and try to not break yours.
Do you want to add this to your bundle or do you prefer I create a new one "dpd-fr-pickup" ?

I may add another codes later, but I wanted to know what do you prefer :)

### CODE REVIEW

File: src/ValueObject/PUDO.php [hunk 1]
```diff
@@ -12,6 +12,8 @@ class PUDO
     public bool $active;
     public Type $type;
     public string $language;
+    public ?string $name;
+    public ?int $ditance;
```
Comment (mglowala) [inline src/ValueObject/PUDO.php]: Typo?

**Review by mglowala — CHANGES_REQUESTED** [review]

File: src/ValueObject/PUDO.php (hunk 1 above)
Comment (Snowbaha) [reply src/ValueObject/PUDO.php]: What do you mean?
Brut XML is:
`<PUDO_ID>P66986</PUDO_ID>
            <DISTANCE>336</DISTANCE>
            <NAME>SAKADO</NAME>
            <ADDRESS1>8 rue Fecauderie</ADDRESS1>`

**Comment by Snowbaha** [comment]
you can see the required parameters on the left column. The last 5 are required but not used -_- 
image

**Comment by Snowbaha** [comment]
Hello, what do you think about the changes?
Maybe it will be better to create an abstract PUDOList.php then create a service per country (PUDOListPl and PUDOListFr) but It will introduce some BC so I prefered to add the function "fr".

If you think it's not adpated for your bundle (because of name PL), I can create a new one "dpd-fr-pickup-services-bundle".

 I don't know what is the best solution, let me know do you think, I am open and I can understand If this PR is not good for you.

File: src/ValueObject/PUDO.php (hunk 1 above)
Comment (mglowala) [reply src/ValueObject/PUDO.php]: ditance/distance

Comment (Snowbaha) [reply src/ValueObject/PUDO.php]: Oh ok, it's fixed here : https://github.com/Snowbaha/dpd-pl-pickup-services-bundle/commit/d55142301ef0ed8dd3140812f00cc522ee86fd6d

**Comment by malarzm** [comment]
Closing this. DPD is known for having separate integrations for their markets and I think we'll do more harm than good trying to keep them in one package.

Thanks @Snowbaha for the PR and sorry it took us this long to articulate reasoning

**Pull request closed without merging**